bridge-adapters/kinetic-bridgehub-adapter-generic-rest/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Stop running the script on an error
+set -e
+
+# Build the jar
+docker build -f Dockerfile.builder -t kineticdata/bridgehub-adapter-bmc-ars-rest-builder:latest .
+docker run -v $PWD:/project -w=/project kineticdata/bridgehub-adapter-bmc-ars-rest-builder:latest
```

Comment:
Is the name of this docker image correct?

Reply:
@austintp I replaced what Jack found with `kineticdata/bridgehub-adapter-generic-rest-builder`.  Is that the correct value?

Reply:
Yes